components/FutureGraph.tsx

```diff
@@ -0,0 +1,318 @@
+              fill="#FFD053"
+            />
+          </svg>
```

Comment:
Since we're already in an `<svg>` context and since this `<path>` is just for text, can we switch this to something like (these styles aren't exact but you get the gist):
```suggestion
          <text
            fill="#FFD053"
            x={x - 30}
            y={-78.95 * graphThreshold + 76.84}
            style={{ position: 'relative', marginLeft: '30px' }}
          >
            2000 kW
          </text>
```

Reply:
just to keep this file size somewhat reasonable, can we abstract this out?